weapons.py

```diff
@@ -22,7 +23,7 @@ def __init__(
         self.min_crit = min_crit
         self.ranged = ranged
         self.topple = topple
-
+        self.base = base
```

Comment:
Is this base actually used anywhere? I don't see it

Reply:
I think it got removed in the merge, I'll restore it